packages/components/src/InputGroup/InputGroup.mdx

```diff
@@ -0,0 +1,55 @@
+---
+name: InputGroup
+menu: Components
+route: /components/input-group
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { InputGroup } from ".";
+import { InputText } from "../InputText";
+import { InputTime } from "../InputTime";
+import { CivilTime } from "@std-proposal/temporal";
+
+# InputGroup
+
+<ComponentStatus stage="rc" responsive="no" accessible="yes" />
+
+Group a different kinds fields to show a visual relationship. If a number of
+form fields once submitted make up a group of content (ie. an address, or first
+and last name becoming a full name) they are great candidates for an input
+group.
+
+The default layout is for fields to flow in a vertical direction, one above the
+next. Use the `flowDirection` prop to specify when you want fields to flow in a
+horizontal direction.
+
+**Warning** - Nested `InputGroup` components need to be of
+`flowDirection="horizontal"` to render properly. If you don't follow this, the
+group will not render and en error will appear in the browser console.
+
+<Playground>
+  <InputGroup>
+    <InputText placeholder="Street 1" />
+    <InputText placeholder="Street 2" />
+    <InputGroup flowDirection="horizontal">
+      <InputText placeholder="City" />
+      <InputText placeholder="Province" />
+    </InputGroup>
+    <InputGroup flowDirection="horizontal">
+      <InputText placeholder="Zip code" />
+      <InputText placeholder="Country" />
+    </InputGroup>
+  </InputGroup>
+</Playground>
+
+<Playground>
```

Comment:
Normally more specific examples would come after a `---` following the props table.

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Button/Button.mdx